dd-sdk-android-core/src/test/kotlin/com/datadog/android/core/internal/data/upload/RotatingDnsResolverTest.kt

```diff
@@ -111,6 +114,30 @@ internal class RotatingDnsResolverTest {
         assertThat(result2).containsExactlyElementsOf(fakeInetAddresses)
     }
 
+    @RepeatedTest(30)
```

Comment:
do we need to repeat it 30 times? won't it be too long? we are already spinning 100 threads

Reply:
yes but it's randomly re-produceable. I am afraid that if we lower it it might be a false positive